sirepo/package_data/static/js/raydata.js

```diff
@@ -218,8 +218,34 @@ SIREPO.app.directive('replayPanel', function() {
             modelName: '=',
         },
         template: `
-          <button type="submit" class="btn btn-primary" data-ng-click="">Start Replay</button>
+          <button type="submit" class="btn btn-primary" data-ng-click="startReplay()">Start Replay</button>
```

Comment:
This button should be not visible until the user has supplied the catalogs and numScans. It should also be grayed out while the replay is running (maybe add some spinner indicating things are running too).

Reply:
f69cb0460a842c98b329b6e5cbe841843fbbb762